test/format/snap.c

```diff
@@ -607,7 +670,8 @@ snap_repeat_single(TINFO *tinfo)
     u_int v;
     int count;
 
-    ts = maximum_read_ts();
+    /* Repeat an operation that's before any running operation. */
+    ts = timestamp_maximum_committed();
```

Comment:
Is there any sort of shared code between test_timestamp_abort, test_format, et al? This function exists in a couple of places now.